### PR TITLE
Pure CSS

### PR DESCRIPTION
**Pull request by chrism**
This PR converts all the SCSS to CSS and removes SASS and `ember-cli-sass` completely.

### CODE REVIEW

**Comment by samselikoff** [comment]
Nice work! Looks like there's a lint failure in the Travis tests.

This looks like a backwards-compatible change, correct? Could you write up some upgrade instructions (even in this PR) so they can be easily added when the next release rolls around?

**Comment by chrism** [comment]
Hi @samselikoff sorry for the newb question but how can i see/resolve the travis lint failure?

I ran ember-try locally and everything seemed ok.

As for upgrading I think, in theory, everything should work without any changes because people already using SASS in their apps will be able to import as they already are doing - and people using CSS will now be able to import the CSS files directly.

I cant really envisage a situation where someone will need to alter their files after the update.

Im on holiday right now but will try to get some internet access and have a go fixing this so it can be merged if theres something i need to do. Thanks!

**Comment by samselikoff** [comment]
No prob! First click that "Details" link down there next to the build failure, then you'll end up [here](https://travis-ci.org/yapplabs/ember-modal-dialog/builds/536849498?utm_source=github_status&utm_medium=notification). If you scroll a bit you'll see 

image

which shows the `yarn lint:js` command failed

**Comment by samselikoff** [comment]
I guess only upgrade notes would be to tell people they can remove ember-cli-sass if this is the only reason they were installing it, which is unlikely... but we could put a note about it somewhere

**Comment by chrism** [comment]
Thanks @samselikoff.

I've fixed the lint issue and added some upgrading text to the readme now.

Hopefully Travis should pass now.

**Comment by samselikoff** [comment]
LGMT! @lukemelia ?

**Comment by samselikoff** [comment]
think its a pretty simple change

**Comment by lukemelia** [comment]
Thanks for your work on this @chrism.

Feel free to merge @samselikoff. Otherwise, I'll try to get to it over the weekend.

**Comment by chrism** [comment]
😁

**Comment by Turbo87** [comment]
just FYI, the README is now somewhat misleading for SASS users due to https://stackoverflow.com/questions/7111610/import-regular-css-file-in-scss-file/36166487#36166487

tl;dr if you `@import` the CSS files in SASS you should drop the `.css` file extension, otherwise the files won't be imported correctly.

**Comment by chrism** [comment]
I’d hoped that is pretty clearly explained in the documentation here...

https://github.com/yapplabs/ember-modal-dialog/blob/master/README.md#configuring-styles

Where I even included both examples with and without the suffix?

**Comment by Turbo87** [comment]
wow, I must have been blind 😂 

sorry about the noise! 🤐

**Comment by chrism** [comment]
Its all good 😁👍
